store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonReaderBuilder.java

```diff
@@ -61,6 +64,33 @@ public CarbonReaderBuilder projection(String[] projectionColumnNames) {
     return this;
   }
 
+  /**
+   * All Columns need to be Projected
+   *
+   * @return CarbonReaderBuilder object
+   * @throws IOException
+   */
+  public CarbonReaderBuilder projectAllColumns() throws IOException {
```

Comment:
can you add a description for this method in reader document

Reply:
ok, done